consensus/istanbul/backend/snapshot.go

```diff
@@ -156,7 +156,7 @@ func (s *Snapshot) apply(headers []*types.Header, gov governance.Engine, addr co
 	snap := s.copy()
 
 	// Copy values which might be changed by governance vote
-	snap.Epoch, snap.Policy, snap.CommitteeSize = getGovernanceValue(gov, snap.Number)
+	snap.Epoch, snap.Policy, snap.CommitteeSize = effectiveParams(gov, snap.Number+1)
```

Comment:
This line appears to be redundant. As it has already been merged, please take it into consideration at a later time.